bigbluebutton-config/bin/bbb-conf

```diff
@@ -1456,6 +1484,23 @@ if [ $CHECK ]; then
      exit 0
 fi
 
+#
+# Return a list of the running meetings
+#
+if [ $RUNNING_MEETINGS ]; then
+    xml=$( get_meetings_xml )
+    has_any=$( echo $xml | grep "no meetings were found on this server" )
+    if [ "$has_any" == "" ]; then
+        if [ $HUMAN_READABLE ]; then
+            list_of_internalId=$( ruby /usr/local/bigbluebutton/core/lib/xml_parser.rb -x "$xml" --human-readable)
```

Comment:
If we're going to use an external helper program in a different language, it should probably make the http request itself, instead of making the http request in the bash script and passing the data.